docs/organizations/managing-people.md

```diff
@@ -0,0 +1,66 @@
+# Managing people
+
+Members of an organization can see the details of the repositories in that organization and add new repositories to be analyzed by Codacy. Besides this, **Codacy only analyzes commits to repositories from contributors who are members of the corresponding organization on Codacy**.
+
+!!! important
+    Make sure that your team members join your organization on Codacy so that Codacy analyzes their commits in the repositories of that organization.
+
+!!! note
+    The organizations on Codacy that contain your personal repositories can only have the owner of the repositories as their single member.
+
+    **If you're using Bitbucket Cloud** the organization that contains your personal repositories can have other members since Bitbucket Cloud doesn't have a different type of [workspace](https://support.atlassian.com/bitbucket-cloud/docs/what-is-a-workspace/){: target="_blank"} for personal use.
+
+To list and manage the members of your organization open your organization **Settings**, page **People**:
+
+![People in an organization](images/organization-people.png)
+
+On this page you can see the list of members of your organization and when they last logged in to Codacy.
+
+<!-- TODO How to see if members are "regular members" or organization owners? -->
+
+## Joining an organization
```

Comment:
Ok, here you could mention the BB special behaviour for personal organizations 👀